bb-new.html

```diff
@@ -0,0 +1,282 @@
+              <div class="dropdown">
+                <a href="#services">Services</a>
+                  <div class="dropdown-content">
+                    <a href="#access">Access</a>
+                    <a href="#defend">Defend</a>
+                    <a href="#implement">Implement</a>
+                    <a href="#support">Support & Training</a>
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#industries">Industries</a>
+                  <div class="dropdown-content">
+                    <a href="#financial">Financial Services</a>
+                    <a href="#healthcare">Healthcare</a></li>
+                    <a href="#professional">Professional Services</a>
+                    <a href="#education">Education</a>
+                    <a href="#government">Government</a>
+                    <a href="#utilties">Utilities & Manufactoring</a>
+                    <a href="#transporation">Transportation</a>
+                    <a href="#consumer">Consumer industries</a>
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#partners">Partners</a>
+                  <div class="dropdown-content">
+                    <a href="#financial">Financial Services</a>
+                    <a href="#healthcare">Healthcare</a></li>
+                    <a href="#professional">Professional Services</a>
+                    <a href="#education">Education</a>
+                    <a href="#government">Government</a>
+                    <a href="#utilties">Utilities & Manufactoring</a>
+                    <a href="#transporation">Transportation</a>
+                    <a href="#consumer">Consumer industries</a>
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#blog">Blog</a>
+                  <div class="dropdown-content">
+                    <a href="#enterprise">Enterprise Blog</a>
+                    <a href="#developer">Developer</a></li>
+                    <a href="#support">Support Blog</a>
+                  </div>
+              </div>
+              <div class="topnav-right">
+                <img src="/Users/berangerelam/Desktop/images-blackberry/search.png">
+              </div>
+              <div class="dropdown topnav-right">
+                <a href="#contact">Telephone icon</a>
+                  <div class="dropdown-content">
+                    <a href="#register">Register for updates</a>
+                    <a href="#sales">Contact Sales</a></li>
+                    <a href="#contactus">Contact Us</a>
+                  </div>
+                </div>            
+            </div>
+          </header>         
+        </div>         
+      </div>
+      <div class="row">
+        <div class="column"> 
+            <img src="/Users/berangerelam/Desktop/images-blackberry/windmill.jpg" alt="several windmills in a line">
```

Comment:
All images should be placed in a folder within html-project, and then referenced as a relative src here.
This src is not accessible to anyone beyond your own Desktop.

Example:
- images are within /images folder
- tag would be: `<img src="./images/windmill.jpg" />`